app/src/main/AndroidManifest.xml

```diff
@@ -434,6 +434,9 @@
       </intent-filter>
     </activity>
 
+    <activity android:name=".ui.activities.SettingsNewActivity" />
```

Comment:
Let's alphabetize this (I know it will eventually be renamed to `SettingsActivity`.)